api/db/migrations/20200625113402_add-column-email-to-organizations.js

```diff
@@ -0,0 +1,14 @@
+const TABLE_NAME = 'organizations';
+const TITLE_COLUMN = 'email';
```

Comment:
petit renommage possible:

```suggestion
const EMAIL_COLUMN = 'email';
```

Reply:
ou encore:

```suggestion
const COLUMN_NAME = 'email';
```